src/current/v22.2/sso-db-console.md

```diff
@@ -51,7 +63,7 @@ The following OIDC [cluster settings](cluster-settings.html) are used to configu
 
 - `enabled` is a Boolean that enables or disables SSO.
 - `client_id` and `client_secret` are generated by the external identity provider.
-- `redirect_url` specifies the callback URL that redirects the user to CockroachDB after a successful authentication. This can be the address of a node in the cluster or the address of a load balancer that routes traffic to the nodes. The path must be appended with `/oidc/v1/callback`.
+- `redirect_url` Specifies the callback URL that redirects the user to CockroachDB after a successful authentication. This can be the address of a node in the cluster or the address of a load balancer that routes traffic to the nodes. You must append the path with `/oidc/v1/callback`. Accepts either a single URL string (for example, `https://{ your cluster's domain }:8080/oidc/v1/callback`) or a JSON object with the following format, which supports region-based OIDC authentication with a callback URL configured per region: <br />{{ oidc_authentication_redirect_url_json }}
```

Comment:
```suggestion
- `redirect_url` specifies the callback URL that redirects the user to CockroachDB after a successful authentication. This can be the address of a node in the cluster or the address of a load balancer that routes traffic to the nodes. You must append the path with `/oidc/v1/callback`. Accepts either a single URL string (for example, `https://{ your cluster's domain }:8080/oidc/v1/callback`) or a JSON object with the following format, which supports region-based OIDC authentication with a callback URL configured per region: <br />{{ oidc_authentication_redirect_url_json }}
```